test/feasibility.jl

```diff
@@ -25,34 +25,152 @@ function test_no_solution()
     model = Model()
     @variable(model, x, Bin)
     # do not support binary
-    @test_throws ErrorException ModelAnalyzer.Feasibility.dual_feasibility_report(
+    @test_throws "No primal" ModelAnalyzer.analyze(
+        ModelAnalyzer.Feasibility.Analyzer(),
         model,
-        Dict(),
     )
     # no dual solutions available
-    @test_throws ErrorException ModelAnalyzer.Feasibility.dual_feasibility_report(
+    # @test_throws ErrorException 
+    data = ModelAnalyzer.analyze(
+        ModelAnalyzer.Feasibility.Analyzer(),
+        model,
+        primal_point = Dict(JuMP.index(x) => 1.0),
```

Comment:
@odow this might be slightly bad for user.
We could try allowing `Dict(x => 1.0)`: either intercepting the specific kwarg or some sort of `map_variables`